app/lib/settings/inputs/pen.dart

```diff
@@ -91,6 +91,20 @@ class PenInputSettings extends StatelessWidget {
                               );
                             },
                           ),
+                          InputMappingListTile(
+                            inputName: AppLocalizations.of(context).invertedPen,
+                            currentValue: config.invertedPen,
+                            defaultValue: InputMappingDefault.invertedPen,
+                            icon: const PhosphorIcon(PhosphorIconsLight.pen),
```

Comment:
Maybe a better icon? Otherwise we have the same icon for both settings

Reply:
We could add a transform.rotate
Or alternativly look for a different icon in https://phosphoricons.com/

Reply:
Or flipping? I'm currently not really sure how, but i think practically invertedStylus is just eraser input at the top of the stylus

Reply:
Did you try to use the transform.flip widgey with the current icon widget as child? Didnt it work?

Reply:
I tried, but InputMappingListTile expects an Icon type rather than a generic widget, so it complained if I tried to pass in the transform of the icon.

Reply:
This is our own widget. free to change the type to Widget :)

I think its located in app/lib/widgets

Reply:
Great! It's done now, I think flipping it over x and y looks pretty good.
<img width="212" alt="image" src="https://github.com/user-attachments/assets/908f7c78-d165-4d75-9240-156cfd0ee0b2" />

Reply:
Looks good!